.github/workflows/quick.yaml

```diff
@@ -94,29 +101,43 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        os: [ ubuntu-22.04 ]
+        os:
+          - ubuntu-22.04
+        compiler:
+          - { CC: gcc, CXX: g++ }
+          - { CC: clang, CXX: clang++ }
+        layer:
+          - { name: layer-00-default, nick: default }
+          - { name: layer-01-minimal, nick: minimal }
+          - { name: layer-02-maximus, nick: maximus }
 
     runs-on: ${{ matrix.os }}
 
```

Comment:
> PR description: `Backport of PR #1917`

Please use commit SHAs instead of (or, if really needed, in addition to) GitHub PR numbers in such references:

* Git commands understand commit SHAs but do not understand PR numbers.
* PR numbers are GitHub-specific and, hence, more volatile (e.g., imagine us migrating repository to GitLab someday).
* Using GitHub PR numbers creates more noise. I am not saying that _any_ PR number reference is noise, but using PR numbers in commit messages (that are often ported to other GitHub repositories) does create noise in some cases because GitHub tracks such events and creates unwanted notifications.
* On GitHub, both SHAs and commit numbers provide easy access to the right change.